web-animations-2/Overview.bs

```diff
@@ -2855,3 +2855,21 @@ Add:
 >     </div>
 >
 >     Note: Need to spec this properly somewhere.
+
+The <code>AnimationPlaybackEvent</code> interface {#the-animationplaybackevent-interface}
+
+Replace double with CSSNumberish as the type for currentTime and timelineTime.
+
+<pre class='idl'>
+[Exposed=Window]
+interface AnimationPlaybackEvent : Event {
+    constructor(DOMString type, optional AnimationPlaybackEventInit
+    eventInitDict = {});
+    readonly attribute CSSNumberish? currentTime;
+    readonly attribute CSSNumberish? timelineTime;
+};
+dictionary AnimationPlaybackEventInit : EventInit {
+    CSSNumberish? currentTime = null;
+    CSSNumberish? timelineTime = null;
+};
+</pre>
```

Comment:
Given that web-animations-1 proceeds to use the type in the attributes and dictionary members we might need to repeat the section here.

Reply:
done

Reply:
In making the change, I noticed that rules for formatting attributes, methods, et al. where missing.  Copied this rules for web-animations-1.